### PR TITLE
List

### PR DESCRIPTION
**Pull request by nishasen**
image
image
image

### CODE REVIEW

**Review by krishanu2911 — APPROVED** [review]
No changes required 
Nice use of CSS variable
Good class naming easy to understand

File: HTML_Pages/list.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                    <li class="list-item">Single-line item</li>
+                    <li class="list-item">Single-line item</li>
+                    <li class="list-item">Single-line item</li>
+                </ul>
+            </div>
+        </section>
+        <section class="comp-code-box">
+            <pre>
+                <code class="html">
+
```
Comment (sadathshariff) [inline HTML_Pages/list.html:75]: You haven't added any code here

Comment (nishasen) [reply HTML_Pages/list.html:75]: Yes will add that after PR review, this basically contains the code that user will see

**Review by sadathshariff — APPROVED** [review]
Looks clean and good Naming conventions

**Pull request merged**